pkg/servers/identity/metrics.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Akamai Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package identity
+
+import (
+	"go.opentelemetry.io/otel"
+)
+
+const meterName = "linode/linode-cosi-driver/servers/identity"
+
+func (s *Server) registerMetrics() error {
```

Comment:
Does this need to return an error?

Reply:
Ok can you add a TODO or something then, just so it's clear that there's a rationale for it for anybody who comes looking before then.